azure-pipelines/Master/BuildAndPushPackage.yml

```diff
@@ -68,17 +68,17 @@ stages:
       steps:
 
       - task: DotNetCoreCLI@2
-        displayName: dotnet pack "src/${{ project.value }}/${{ project.value }}.csproj"
+        displayName: dotnet pack "${{ project.value }}"
         inputs:
           command: pack
           packagesToPack: "src/${{ project.value }}/${{ project.value }}.csproj"
           versioningScheme: byBuildNumber
           arguments: --configuration Release
 
       - task: NuGetCommand@2
-        displayName: nuget push "src/${{ project.value }}/${{ project.value }}.csproj"
+        displayName: nuget push "${{ project.value }}"
         inputs:
           command: push
-          packagesToPush: "src/${{ project.value }}/**/*.nupkg;!src/${{ project.value }}/**/*.symbols.nupkg"
+          packagesToPush: "**/*.nupkg;!**/*.symbols.nupkg"
```

Comment:
Looks to me like this pushes all packages, rather than the one that was packed in the last step. Is that accurate, and is that what we want?

Reply:
Note that the previous step only builds packages for the project specific to the current job. Remember, this is a template: job-per-project.

Reply:
(There are multiple packages built per project.)

Reply:
(And the workspace is clean, because we get a fresh VM per job.)

Reply:
> And the workspace is clean

That's what I was most worried about :+1: